packages/desktop-client/src/components/reports/ReportTopbar.tsx

```diff
@@ -60,6 +65,20 @@ export function ReportTopbar({
     defaultItems(cond);
   };
 
+  const downloadSnapshot = async () => {
+    const reportElement = document.getElementById('custom-report-content');
+    const title = report.name;
+    if (reportElement) {
+      const dataUrl = await toPng(reportElement);
+      const link = document.createElement('a');
+      link.href = dataUrl;
+      link.download = `${getToday()} - ${title}.png`;
```

Comment:
Can be replaced with monthUtils

```suggestion
      link.download = `${monthUtils.currentDay()} - ${title}.png`;
```